README.md

```diff
@@ -86,13 +102,15 @@ A Refactoring or Remodeling intends to alter the program in some way without cha
 
 | Code | Known Approaches |
 | --- | --- |
-| `r - ` | One of: <ul><li>Provable refactoring<sup>[2]</sup></li><li>Test-supported Procedural Refactoring<sup>[3]</sup> entirely within test code</li></ul> |
-| `R - ` | Test-supported Procedural Refactoring<sup>[3]</sup> |
-| `R!!` | Identified single, named refactoring, but executed by editing code or without whole-project test coverage. |
-| `R**` | Remodeled by editing code, even in small chunks. |
+| `. R` | One of: <ul><li>Provable refactoring<sup>[2]</sup></li><li>Test-supported Procedural Refactoring<sup>[3]</sup> entirely within test code</li></ul> |
+| `- R` | Test-supported Procedural Refactoring<sup>[3]</sup> |
+| `! R` | Identified single, named refactoring, but executed by editing code or without whole-project test coverage. |
+| `* R` | Remodeled by editing code, even in small chunks. |
```

Comment:
The switch from * to @ was missed on this line

Reply:
Fixed. Thanks!